cfgov/jobmanager/tests/test_blocks.py

```diff
@@ -156,13 +150,30 @@ def test_html_has_job_listings(self):
         self.assertIn('Apr. 21, 2099', html)
         self.assertEqual(html.count('Silicon Valley'), 2)
 
+    def test_is_efficient(self):
```

Comment:
I love this practice of keeping an eye on query count.

Reply:
Same! I wonder if we could make a general pattern out of this...